app/controllers/root_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 class RootController < ApplicationController
-  def index; end
+  def index
```

Comment:
You can have `ArticlesController` at `/articles` and add `root to: 'articles#index'` to your `config/routes.rb`. That's how it is usually done

Reply:
In next PR will be the article CRUD